exporters/trace/zipkin/build.gradle

```diff
@@ -7,6 +7,7 @@ description = 'OpenCensus Trace Zipkin Exporter'
 
 dependencies {
     compile project(':opencensus-api'),
+            libraries.guava,
```

Comment:
odd that this is needed here..

Reply:
I think this should be scrubbed as the following isn't useful enough to claim the dep

```bash
$ ag google src/main/
src/main/java/io/opencensus/exporter/trace/zipkin/ZipkinTraceExporter.java
19:import static com.google.common.base.Preconditions.checkState;
21:import com.google.common.annotations.VisibleForTesting;

src/main/java/io/opencensus/exporter/trace/zipkin/ZipkinExporter.java
19:import com.google.common.annotations.VisibleForTesting;
```

Reply:
by scrub I mean throw IllegalStateException directly and just comment // VisibleForTesting

Reply:
I think this is covered here:
https://github.com/census-instrumentation/opencensus-java/issues/1113